public/src/js/widgets/columns/story-package.js

```diff
@@ -173,6 +173,17 @@ export default class Front extends ColumnWidget {
         if (item.meta.snapType()) {
             return 'You cannot add snaps to packages';
         }
+
+        if (item.group.parentType === 'Collection') {
```

Comment:
do we need the if? When we get here is it possible to have something other that parentType Collection?